[rhevm][hyperv] tc_1052  failed due to unable to search the hypervisorid in satellite ui

Error messages:
```
2021-03-17 15:33:04 [INFO] >>>step3: run virt-who with hypervisor_id=hwuuid
2021-03-17 15:33:05 [INFO] Succeeded to add option hypervisor_id=hwuuid
2021-03-17 15:33:20 [INFO] Start to run virt-who by service
2021-03-17 15:33:48 [INFO] pending_job: 0, is_429: no, loop_num: 0, loop_time: -1, send_num: 1, error_num: 0, thread_num: 1
2021-03-17 15:33:48 [INFO] virt-who is terminated by expected_send and expected_loop ready
2021-03-17 15:34:06 [INFO] Succeeded to get satellite host_id: 10.66.144.13:1569
2021-03-17 15:34:06 [INFO] virtwho thread number(1) is expected
2021-03-17 15:34:06 [INFO] virtwho error number(0) is expected
2021-03-17 15:34:06 [INFO] virtwho send number(1) is expected
2021-03-17 15:34:06 [INFO] Finished to validate all the expected options
2021-03-17 15:34:06 [INFO] Succeeded to search hypervisorId:76024700-8BAD-11E2-B2B0-6C3BE51D8780
2021-03-17 15:34:08 [INFO] Succeeded to get satellite host_id: 10.66.144.13:1569
2021-03-17 15:34:09 [INFO] Succeed to get expected host info
2021-03-17 15:34:09 [ERROR] Failed to search hypervisorDisplay:76024700-8BAD-11E2-B2B0-6C3BE51D8780
2021-03-17 15:34:11 [INFO] Succeeded to delete option hypervisor_id
2021-03-17 15:34:12 [INFO] Succeeded to get satellite host_id: 10.66.144.13:1569
2021-03-17 15:34:15 [INFO] Succeeded to delete host: 10.66.144.13
2021-03-17 15:34:15 [ERROR] Failed step: step3
2021-03-17 15:34:15 [ERROR] Failed to run case, please check the failed steps

```

--- a/tests/tier1/tc_1052_check_hypervisor_id_option_in_etc_virtwho_d.py
+++ b/tests/tier1/tc_1052_check_hypervisor_id_option_in_etc_virtwho_d.py
@@ -62,7 +62,7 @@ class Testcase(Testing):
                 host_display = self.stage_consumer_get(self.ssh_host(), register_config,
                                         host_name, host_uuid)
                 host_display_name = host_display['name']
-            if hypervisor_display in host_display_name:
+            if hypervisor_display or hypervisor_display.lower() in host_display_name:
                 logger.info("Succeeded to search hypervisorDisplay:{0}".format(hypervisor_display))
                 results.setdefault(step, []).append(True)
             else:

